src/collectors/windows.plugin/perflib-hyperv.c

```diff
@@ -0,0 +1,1553 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#include "windows_plugin.h"
+#include "windows-internals.h"
+
+#define _COMMON_PLUGIN_NAME "windows.plugin"
+#define _COMMON_PLUGIN_MODULE_NAME "PerflibHyperV"
+#include "../common-contexts/common-contexts.h"
+
+#define HYPERV  "hyperv"
+
+
+long chart_priority = NETDATA_CHART_PRIO_WINDOWS_HYPERV;
+
+void get_and_sanitize_instance_value(PERF_DATA_BLOCK *pDataBlock,  PERF_OBJECT_TYPE *pObjectType, PERF_INSTANCE_DEFINITION *pi, char *buffer, size_t buffer_size)
```

Comment:
I loved this function, I would suggest to move it to `windows_plugin.h` as static inline, or to windows.plugin main file.